branding/pdf/entities.adoc

```diff
@@ -2,8 +2,9 @@
 :ibmz: IBM Z
 :zsystems: {ibmz}
 :zseries: {ibmz}
-:ppc: POWER
-:ppc64le: IBM Power PC
+:ppc: IBM Power
```

Comment:
No, this is wrong. You need to check whether we are talking about ppc/ppc64/ppc64le in a technical way (the same way we say "you need 8 GB of RAM, 200 GB of disk space", i. e. precise details) or in a more "marketing" way (the same way we say "you need an Intel-compatible server, you need a network connection", i. e. more generic way).

If the former, then we must use ppc, ppc64 or ppc64le.

If the latter, then we should use "IBM POWER".

That's why I was saying we need to review accurately where were we using the placeholders and for what, we cannot just replace the entities and the tables.

For this case:

```suggestion
:ppc: ppc
```

Reply:
There's no point using an entity for this, though. If we want to write "ppc" we can just write it.

Reply:
OK